packages/api/app/Commands/AlgoliaIndex.js

```diff
@@ -60,11 +64,8 @@ class AlgoliaIndex extends Command {
 				const ownerUser = item.users.find((user) => user.pivot.role === roles.OWNER);
 				const tec = {
 					...item,
-					category: item.terms.find(
-						(term) =>
-							term.taxonomy.taxonomy === CATEGORY_TAXONOMY_SLUG &&
-							term.parent_id === null,
-					).term,
+					...normalizeAlgoliaTechnologyTerms(item),
```

Comment:
@luizeboli , tava dando um conflito com o arquivo da master. Fiz o merge aqui pra resolver, mas não fui testar. Só verifica aqui se é isso mesmo e, se for, pode fazer o merge direto...já vou deixar aprovado 👍